### ISSUE TITLE
Should timeliness dashboard show date of first publication?

### ISSUE THREAD

**Issue by andylolz** [IATI/IATI-Dashboard#527]
It’s usually not possible to compute an organisation’s timeliness frequency without knowing when they first published, since this determines how the monthly publication numbers are interpreted.

It would be useful to show this date in the table and corresponding CSV.

The fact the dashboard no longer runs daily might cause an issue here, since the date could be a couple of days out.

**Comment by samuele-mattiuzzo** [comment]
I like this, I'm actually investigating what pieces of information we could/should expose that would make it a bit more straight forward to debug and investigate issues

**Comment by andylolz** [comment]
I think this is the same as (or very similar to) #336.

**Comment by stevieflow** [comment]
https://dashboard.iatistandard.org/publishing-statistics/timeliness-frequency/ does now have a column for "first published", but this is a set of bands, rather than a specific date

@Bjwebb what is the data source for these values?

![Image](https://github.com/user-attachments/assets/2156a23a-1fa9-4593-b980-1789f25bb70a)

**Comment by andylolz** [comment]
That’s right – this was part of codeforIATI analytics. There’s a bit more detail about it in https://github.com/codeforIATI/analytics/issues/33.

**Comment by stevieflow** [comment]
Thanks @andylolz 

Excuse my pun, but https://github.com/codeforIATI/analytics/issues/33 was ``3 to 5 years ago`` :)

**Comment by emmajclegg** [comment]
This is related to #314 

'First published date' is not available in either the Timeliness tables in the interface, or in the associated CSV export. Instead, a categorical label is used ("More than 5 years ago", "3-5 years ago", etc.).

> @Bjwebb what is the data source for these values?

Understanding the source ^^ would be a first step, then we can decide whether to replace the category label with a specific date.
